packages/devtools_app/lib/src/inspector/inspector_service.dart

```diff
@@ -968,11 +1122,73 @@ class ObjectGroup {
       'arg': node.dartDiagnosticRef.id,
       'subtreeDepth': subtreeDepth.toString(),
     };
-    return parseDiagnosticsNodeDaemon(invokeServiceMethodDaemonParams(
+    final json = await invokeServiceMethodDaemonParams(
       'getDetailsSubtree',
       args,
+    );
+    return parseDiagnosticsNodeHelper(json);
+  }
+
+  Future<void> invokeSetFlexProperties(
```

Comment:
These methods were all just moved from the separate obsolete extension method file.